react/features/base/dialog/components/Dialog.web.js

```diff
@@ -56,6 +56,7 @@ class Dialog extends AbstractDialog {
                 width = { this.props.width || 'medium' }>
                 <div>
                     <form
+                        className = 'modal-dialog-form'
                         id = 'modal-dialog-form'
```

Comment:
Given that the `id` is required (I guess by `<AKButton form = 'modal-dialog-form'>`) why is a className necessary?